tests/benchdnn/inputs/reorder/test_reorder_bfloat16

```diff
@@ -1,6 +1,6 @@
-# f32, s8, u8 <--> bf16
+# f32, bf16, s8, u8 <--> bf16
 --reset
---sdt=f32,s8,u8,f8_e5m2,f8_e4m3 --ddt=bf16
+--sdt=f32,bf16,s8,u8,f8_e5m2,f8_e4m3 --ddt=bf16
```

Comment:
This change seems redundant as bf16->f32 is already covered line 14.

Reply:
Yes, I checked the list and that was missing. I believe it's good to have.